### DISCUSSION TITLE
Mulighet for å duplikere eksisterende tavler

### DISCUSSION THREAD

**Discussion by pernilak** [entur/tavla#311] [Ideas]
I fremtiden ønskes det at Tavla blir tatt i bruk av aktører som AtB og Skyss, som ofte ønsker å vedlikeholde flere ulike tavler. Det kan tenkes at alle disse tavlene skal kunne ha noe lik styling i form av theme og logo. Da kan det være tidkrevende arbeid og fikse disse samme innstillingene om og om igjen for hver enkelt tavle.

For å møte det behovet burde det være lettere å kunne opprette en tavle som allerede inneholder denne stylingen ved opprettelse. Det kan være mange måter å gjør det på, men en måte kan være ved å utnytte "Mine Tavler". Det. burde være enkelt å kunne opprette en tavle, og lagre ønsket branding. På Mine Tavler kunne det f.eks. gått an å lagret denne tavlen som en form for mal, og når man lager ny tavle, kan man heller lage en ny tavle ved å duplikere innstillingene denne tavlen har. Etter som det skal lages funksjonalitet for å endre koordinater fra Admin-panelet, ville ikke lokasjon være noe problem.

Da slipper man jobben med å endre logo, beskrivelse og theming for hver eneste tavle. 

Hvis en slik funksjonalitet skal lages, burde det være synlig hvordan den kan utnyttes for å hjelpe til i prosessen med enkelt opprette masse tavler, slik at jobben for kollektivselskapene eller andre aktører blir enklere!

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
